azkaban-common/src/main/java/azkaban/metrics/CommonMetrics.java

```diff
@@ -34,6 +34,8 @@
   private final MetricsManager metricsManager;
   private Meter dbConnectionMeter;
   private Meter flowFailMeter;
+  private Meter dispatchFailMeter;
```

Comment:
Oh cool. @jamiesjc, In that case, should we just retain these Meters in `ExecutorManager` then instead? The existing metrics can be refactored separately in a different PR.